src/ReactiveDomain.Policy.Tests/ReactiveDomain.Policy.Tests.csproj

```diff
@@ -9,14 +9,14 @@
     </ItemGroup>
   
     <ItemGroup>
-      <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-      <PackageReference Include="xunit" Version="2.4.1" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-      <PackageReference Include="xunit.runner.console" Version="2.4.1">
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
13.0.3 is the latest version, released on March 8th of this year. Can we update to that version?